xija/gui_fit/app.py

```diff
@@ -243,6 +243,18 @@ def __init__(self, model, main_window):
         main_box.addWidget(QtWidgets.QLabel("Timestep: {:.1f} s".format(model.dt)))
         main_box.addWidget(QtWidgets.QLabel("Evolve Method: Core {}".format(model.evolve_method)))
         main_box.addWidget(QtWidgets.QLabel("Runge-Kutta Order: {}".format(4 if model.rk4 else 2)))
+        for msid in self.model.limits:
+            limits = self.model.limits[msid]
+            units = limits["unit"]
+            main_box.addWidget(QtWidgets.QLabel(f"{msid.upper()} limits:"))
+        for limit, val in limits.items():
```

Comment:
Shouldn't this loop be in the `for msid ..` block? Otherwise the `limits` and `units` are just the ones for the last `msid`. I think in practice with the current xija spec files this won't make a difference, but still it doesn't look right to me.